CUJExecutor/manipulateProfilesAndTest

```diff
@@ -19,36 +19,34 @@
 # Compiles aGMM according to the COMPILATION_FLAG and then measures the CUJ specified by the INPUT_FILE (as specified by executeCUJ).
 # Stores the following files at /profile_test_data/<INPUT_FILE_NAME>/<COMPILATION_FLAG>/<DATE>/:
 # - (command_summary.txt) a log of each run command and their output.
-# - (short_summary.txt) a formatted summary including the profile used for compilation, resulting app size and measurements of the POST_CUJ (acquired via a call to executeCUJ). A copy is stored in /tmp for the callee.
+# - (short_summary.txt) a formatted summary including the profile used for compilation, resulting app size and measurements of the MEASURED_CUJ (acquired via a call to executeCUJ). A copy is stored in /tmp for the callee.
 # - (saved_profile.prof) the profile used to compile the app (if applicable). 
 # - (median_clip.mp4) a recording of the median iteration (if requested).
 #
-# @param INPUT_FILE (Formatted with the following on consecutive lines: PRE-CUJ, POST_CUJ, NUM_ITERS, and optionally RECORD (a line with an "r" to indicate that you'd like to record). See CUJS/ for an example file.
-# @param COMPILATION_FLAG (e - empty profile, p - compile actions in Pre-CUJ, c - compile entire CUJ, a - compile all code).
+# @param INPUT_FILE (Formatted with the following on consecutive lines: PRE-CUJ, MEASURED_CUJ, POST_CUJ, NUM_ITERS, and optionally RECORD (a line with an "r" to indicate that you'd like to record). See CUJS/ for an example file.
+# @param COMPILATION_FLAG (e - empty profile, m - compile measured CUJ, u - compile with a cloud profile, f- compile first action, p - compile actions in Pre-CUJ, c - compile pre and measured CUJS, a - compile all code).
+#  	 Optionally, if f, p, c, or m are followed by a number, n, (e.g. m12), that portion of the CUJ will be run through n times during the creation of the profile. If a, e or u are followed by a number, that number will be ignored.
```

Comment:
No action requested.  Pretty neat, flexible, and powerful (e.g. m12).  But very succinct.